src/sporemodder/view/editors/SpuiEditor.java

```diff
@@ -1288,6 +1352,9 @@ private void exportBlocks() throws Exception {
 	private void importSpui() throws Exception {
 		FileChooser chooser = new FileChooser();
 		chooser.setInitialDirectory(getFile().getParentFile());
+		chooser.getExtensionFilters().add(new ExtensionFilter("All SPUI files (*.spui, *.spui_part)", "*.spui", "*.spui_part"));
+		chooser.getExtensionFilters().add(new ExtensionFilter("Partial SPUI files (*.spui_part)", "*.spui_part"));
```

Comment:
I was thinking it would help to avoid file name collisions, and to help differentiate between the two in the file tree. It may also help to pave the way for a feature I suggested years ago for OG SporeModder: the ability to "link" one layout in another, to be merged when packing.